File: Python-basics/Conditional-Statements-Advanced_3/Exercise/new_house_3.py
flower = input()
number_of_flowers = int(input())
budget = int(input())

if flower == "Roses":
    final_price = number_of_flowers * 5
    if number_of_flowers > 80:
        final_price -= final_price * 0.1

elif flower == "Dahlias":
    final_price = number_of_flowers * 3.8
    if number_of_flowers > 90:
        final_price -= final_price * 0.15
    
elif flower == "Tulips":
    final_price = number_of_flowers * 2.8
    if number_of_flowers > 80:
        final_price -= final_price * 0.15
    
elif flower == "Narcissus":
    final_price = number_of_flowers * 3
    if number_of_flowers < 120:
        final_price += final_price * 0.15

elif flower == "Gladiolus":
    final_price = number_of_flowers * 2.5
    if number_of_flowers < 80:
        final_price += final_price * 0.2
    
if final_price >= budget:
    print(f"Hey, you have a great garden weith {number_of_flowers} {flower} and {final_price-budget:.2f} leva left.")
else:
    print(f"Not enough money, you need {budget-final_price:.2f} leva more.")

File: Javascript/Javascript_Basics/Conditional_Statements_Advanced_3_Exercise/newHouse_3.js
function newHouse(input) {
    let flower = input[0];
    let num = Number(input[1]);
    let budget = Number(input[2]);
    let result;

    if (flower == 'Roses') {
        result = num * 5;
        if (num > 80) {

            let discount = result * 0.1;
            result -= discount;
        }
    }
    else if (flower == 'Dahlias') {
        result = num * 3.8;
        if (num > 90) {
            let discount = result * 0.15;
            result -= discount;
        }
    }
    else if (flower == 'Tulips') {
        result = num * 2.8;
        if (num > 80) {
            let discount = result * 0.15;
            result -= discount;
        }
    }
    else if (flower == 'Narcissus') {
        result = num * 3;
        if (num < 120) {
            let discount = result * 0.15;
            result += discount;
        }
    }
    else if (flower == 'Gladlolus') {
        result = num * 2.5;
        if (num < 80) {
            let discount = result * 0.2;
            result -= discount;
        }
    }

    if (result > budget) {
        let money_need = (result - budget).toFixed(2);
        console.log(`Not enough money, you need ${money_need} leva more.`)
    }
    else {
        let money_left = (budget - result).toFixed(2);
        console.log(`Hey, you have a great garden with ${num} ${flower} and ${money_left} leva left.`)
    }
}